sweep/test_utils.go

```diff
@@ -0,0 +1,217 @@
+package sweep
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/btcsuite/btcd/chaincfg/chainhash"
+	"github.com/btcsuite/btcd/wire"
+	"github.com/lightningnetwork/lnd/chainntnfs"
+	"github.com/lightningnetwork/lnd/lnwallet"
+	"sync"
+)
+
+var (
+	defaultTestTimeout = 5 * time.Second
+	mockChainIOHeight  = int32(100)
+)
+
+type mockSigner struct {
+}
+
+func (m *mockSigner) SignOutputRaw(tx *wire.MsgTx,
+	signDesc *lnwallet.SignDescriptor) ([]byte, error) {
+
+	return []byte{}, nil
+}
+
+func (m *mockSigner) ComputeInputScript(tx *wire.MsgTx,
+	signDesc *lnwallet.SignDescriptor) (*lnwallet.InputScript, error) {
+
+	return &lnwallet.InputScript{}, nil
+}
+
+// MockNotifier simulates the chain notifier for test purposes.
+type MockNotifier struct {
```

Comment:
I think this can actually be private. In that case since they're just used for mock tests, prob no need to have full `godoc` comments on all the methods below.

Reply:
That is the annoying thing with test code in go. I reuse this mock in the `utxonursery_test.go`. Would love to see a solution to this.